src/js/reducers/todoReducer.js

```diff
@@ -12,19 +12,18 @@ export default function reducer(state={
             id: 2,
             description: 'empty the bin'
         }
-    ]
+    ],
```

Comment:
maybe better would be exporting that literal object to variable, now that code looks illegibly